src/LINEBot/Response.php

```diff
@@ -63,7 +63,7 @@ public function getHTTPStatus()
      */
     public function isSucceeded()
     {
-        return $this->httpStatus === 200;
+        return 200 <= $this->httpStatus && $this->httpStatus <= 299;
```

Comment:
i came up with an idea could be shorter than this line.

`return $this->httpStatus / 100 === 2`

Reply:
That is also ok if `httpStatus` is verified non-zero.
The original code is safer in general.